.github/workflows/upgrade-python-requirements.yml

```
@@ -62,6 +62,10 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ inputs.python_version }}
+      
```

```suggestion
      # With the updated package lxml>5.0, now the system dev packages are necessary 
      # to run pip-compile. Without installing dev packages libxml2-dev and libxslt-dev, 
      # the requirements upgrade job fails
```

Sometimes commit messages and PRs get far away from the place where the change is made so it's good to have comments explaining reasoning when possible.